src/main.rs

```diff
@@ -89,18 +70,20 @@ fn display_board(board: &Vec<bool>, board_width: usize, board_height: usize) {
                 (true, true, true, false) => "▟",
                 (true, true, true, true) => "█",
             };
-            buffer.push(symbol);
             if n % board_width == board_width - 1 {
-                buffer.push("\n");
+                format!("{}\n", symbol)
```

Comment:
The `format!` macro is nice! Same syntax as `println!` but returns you a string.